_pkgdown.yml

```diff
@@ -0,0 +1,49 @@
+url: https://lintr.r-lib.org
+template:
+  bootstrap: 5
+
+  includes:
+    in_header: |
+      <script defer data-domain="lintr.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
+
+reference:
+- title: Linting
+  contents:
+  - lint_dir
+  - lint_file
```

Comment:
That's just `lint`, right?

Reply:
What do you think of documenting `lint_file`, `lint_dir`, and `lint_package` together? They seem to have similar arguments.

Reply:
Sounds good.